server/server.go

```diff
@@ -84,6 +84,8 @@ const (
 	// idAllocPath for idAllocator to save persistent window's end.
 	idAllocPath  = "alloc_id"
 	idAllocLabel = "idalloc"
+
+	pdRecoveringMarkPath = "/pd/recovering-mark"
```

Comment:
Does it need a cluster_id prefix?

Reply:
no need now

Reply:
Will it be better to add a config instead of directly write to etcd?

Reply:
ok for now

Reply:
/cc @nolouch